src/components/HelpDrawer/HelpContent/SpectralProfilerSettingsFittingHelpComponent.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+export class SpectralProfilerSettingsFittingHelpComponent extends React.Component {
+    public render() {
+        return (
+            <div>
+                <h3>Profile Fitting</h3>
+                <p>With the fitting tab, users can fit the observed spectrum with a model profile. Please ensure that there is only one spectral profile in the plot. Fitting multiple observed spectra is not supported. The supported profile functions are:</p>
+                <ul>
+                    <li>Gaussian</li>
+                    <li>Lorentzian</li>
+                </ul>
+                <p>In addtion, an optional continuum profile can be added in the fit, including a constant profile and a linear profile.</p>
+                <h3>Set up initial solution for the fitter</h3>
+                <p>In order to make the profiler fitting engine work properly, a good set of initial solution of the free parameters need to be supplied. CARTA includes an <em>experimental</em> algorithm to analyze the input profile and makes an educated guess of the initial solution. This can be archieved by clicking the &quot;auto detect&quot; button. If there is a prominent continuum emission, please enable the &quot;w/cont.&quot; toggle before clicking the &quot;auto detect&quot; button.</p>     
+                <p>The auto detect function will report how many components are found in the spectrum and visualize them as green boxes in the plot. Their numerical values are provided too. To see the initial values of different components, please use the slider for navigation. New component may be added via the component spinbox. A component may be removed with the &quot;delete&quot; button.</p>
+                <p>Users may use the GUI to define the initial solution <em>manually</em> by entering the values with the text fields, or by using mouse to draw a <em>box</em> on the plot. The height, width, and center position of the box represent the amplitude, FWHM, and center of a model profile, respectively. The lock button can be used to lock a parameter so that the parameter is fixed during the fitting process. The initial solution of a continuum can be defined manually by entering the values with the text fields, or by using mouse to draw a <em>line</em> on the plot (recommended).</p>
```

Comment:
Users -> You

the mouse

The initial solution of a continuum can be defined -> You can define the initial solution of a continuum

the mouse